release-notes-3.4.0.md

```diff
@@ -1,9 +1,20 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-[SCMOD-7343](https://portal.digitalsafe.net/browse/SCMOD-7060) - Audit Service has been updated to support Elasticsearch 7.2.0
+- **SCMOD-7343**: Audit Service has been updated to support Elasticsearch 7.2.0
+- **SCMOD-5860**: Deployed Audit service in Apollo
+- **SCMOD-5877**: Removed generic exception from interface
+- **SCMOD-4930**: Removed cluster name specification
+- **SCMOD-5878**: Fixed package structure for the generated-source jar
+- **SCMOD-7032**: Added feature to support multiple host names for Elastic search
+- **SCMOD-7034**: Support added to use index templates
+- **SCMOD-7585**: Documentation updated for default values
+- **SCMOD-7731**: Audit template name formatted
+- **SCMOD-7628**: Standard logging format changes done
```

Comment:
Poor description of the change, the change was to follow the caf-logging format. You can add a link to the caf-logging project.